djangocms_versioning_filer/admin.py

```diff
@@ -16,3 +18,13 @@ def get_fieldsets(self, request, obj=None):
                     f for f in fieldset[1]['fields'] if f != 'changed_filename'
                 )
         return fieldsets
+
+    def get_urls(self):
+        return [
+               url(r'^operations/upload/check/(?P<folder_id>[0-9]+)/$',
```

Comment:
This block appears to be double indented.

Reply:
Fixed